riscv/src/continuations/bootloader.rs

```diff
@@ -122,6 +147,9 @@ jump compuation_start;
 // to the shutdown routine.
 jump shutdown_start;
 
+shutdown_sink:
+jump shutdown_sink;
```

Comment:
This will let us later assert that the shutdown routine actually completed (by asserting that the PC in the last row points to the sink).